next.config.js

```diff
@@ -1,3 +1 @@
-module.exports = {
```

Comment:
You actually need to leave this in there for it to work on netlify - can you test that?

Reply:
@joelvarty it appears like it's still able to deploy to netlify without `target: "serverless"`.

see here: https://relaxed-panini-96053e.netlify.app/ - preview mode is still weird though